tello Test/test_tello_connection.py

```
@@ -0,0 +1,173 @@
+
+def test_connection_monitoring():
+    """接続監視テスト（バイナリデータフィルタリング効果確認）"""
+    print("\n接続監視テスト（10秒間）")
+    print("このテストでバイナリデータのフィルタリング効果を確認します")
+    print("Webサーバーのログを観察してください...")
+    
+    start_time = time.time()
+    while time.time() - start_time < 10:
+        remaining = 10 - int(time.time() - start_time)
+        print(f"残り {remaining}秒... ", end="\r")
+        await asyncio.sleep(1)
```

Pylint (3.3.7)

[error] 'await' should be used within an async function

(E1142)